doc/content/reference/root-certificates/_index.md

```diff
@@ -7,6 +7,12 @@ This section contains links to common root SSL certificates used in {{% tts%}},
 
 <!--more-->
 
+## Let's Encrypt
+
+Most {{% tts %}} deployments use the Let's Encrypt DST X3 Trust. If using Let's Encrypt to secure your domain, you may download the Let's Encrypt DST X3 Trust file [here](https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt).
```

Comment:
I think it would be smarter to recommend the ISRG Root X1 instead, as that one is valid until 2035, while the cross-signed X3 is only valid until March 2021.